style.css

```diff
@@ -0,0 +1,65 @@
+
+body {
+    margin: 0px;
+    font-family: 'Roboto', sans-serif;
+}
+
+advantages {
+    max-width: 1440px;
+    max-height: 569px;
+}
+
+.advantages-header {
+    left:567px;
+    margin-top: 120px;
+    margin-bottom: 0;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 47px;
+    color: #383838;
+    position: relative;
+}
+
+hr{
+    position: absolute;
```

Comment:
Avoid positioning please! Remove `position` and `left`
add `margin: 20px auto 0;` instead